nodejs/ne_dbops.cc

```diff
@@ -62,16 +62,16 @@ NAN_METHOD(init) {
 
 class AddWorker : public Nan::AsyncWorker {
  public:
-  AddWorker(Nan::Callback *callback, const std::string &ddbPath, const std::vector<std::string> &paths, bool recursive)
+  AddWorker(Nan::Callback *callback, const std::string &ddbPath, const std::vector<std::string> &paths)
```

Comment:
No, no... don't remove the recursive parameter from `ddb add`.

Reply:
My bad 👍